Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -225,16 +160,14 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
         }
     }
 
-    func SELlongPressLocation(recognizer: UILongPressGestureRecognizer) {
-        if recognizer.state == UIGestureRecognizerState.Began {
-            delegate?.browserLocationViewDidLongPressLocation(self)
-        }
+    func textFieldShouldBeginEditing(textField: UITextField) -> Bool {
+        delegate?.browserLocationViewDidTapLocation(self)
+        return false
     }
 
-    func SELtapLocation(recognizer: UITapGestureRecognizer) {
-        if recognizer.state == UIGestureRecognizerState.Ended {
-            editTextFieldListenerView.hidden = true
-            editTextField.becomeFirstResponder()
+    func SELlongPressLocation(recognizer: UITapGestureRecognizer) {
+        if recognizer.state == UIGestureRecognizerState.Began {
```

Comment:
I think I like began better than ended, but since its a change, just want to make sure its what we want.